public/app/plugins/datasource/testdata/runStreams.ts

```diff
@@ -188,7 +188,7 @@ export function runFetchStream(
     data.refId = target.refId;
     data.name = target.alias || 'Fetch ' + target.refId;
 
-    let reader: ReadableStreamReader<Uint8Array>;
+    let reader: ReadableStreamDefaultReader<Uint8Array>;
```

Comment:
`ReadableStreamReader` is now slightly different (it requires `.read()` to be passed a view).

`response.body.getReader()` returns `ReadableStreamDefaultReader`, so I just updated it to use that type here. This is not a runtime change.